### ISSUE TITLE
Peer To Peer Code Review 

### ISSUE THREAD

**Issue by Chimoneg27** [Obimbo07/awesome-books#4]
### Great job with your project, but here are some suggestions that can make it better♻️🙏👍👌✅

✅Wonderful use github workflows
✅Great job removing linters

- Add live demo link to your README.MD file
- Your partner should also collaborate more equally
- Addbook button is not functioning, please make sure this button along with the other buttons work. This will improve your SPA functionality 
- Also report your partners if they are not willing to collaborate, it will save you time.

**Comment by Al-Amin-Khan-Shakil** [comment]
I agree with Chimoneg27, and about the live demo, I will share these benefits of it. When we use the live demo in our readme file users can quickly and easily access and interact with our project without needing to download, install, or configure anything. This can lower the barrier to entry and encourage more people to try out our application.
